.github/workflows/build_test_deploy.yml

```diff
@@ -14,7 +14,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
-      - run: yarn install --frozen-lockfile --check-files
+      - run: yarn install --immutable
```

Comment:
The `--immutable` flag replaces `--frozen-lockfile`. The `--check-files` flag isn't supported anymore and was always of dubious use, it only mattered if you expected your nm files to be corrupted somehow (which Yarn shouldn't let happen).